app/.eslintrc.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
     'react-native/no-unused-styles': 'error',
     'react-native/no-raw-text': 'off',
     'react-native/no-single-element-style-arrays': 'error',
+    'react/jsx-fragments': ['error', 'syntax'],
```

Comment:
why do we need to have both `error` and `syntax` here and not only `error`?

Reply:
We have to choose whether we prefer `<>` or `<React.Fragment>`, the rule has form `"react/jsx-fragments": [<enabled>, <mode>]` https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md

Reply:
it's okay then, thanks for explanation!